samples/CMakeLists.txt

```diff
@@ -7,12 +7,16 @@ project(KinesisVideoWebRTCClientSamples LANGUAGES C)
 message("OPEN_SRC_INSTALL_PREFIX=${OPEN_SRC_INSTALL_PREFIX}")
 
 if (WIN32)
-    if(EXISTS "C:\\gstreamer\\1.0\\x86_64\\bin\\pkg-config.exe")
-      set(PKG_CONFIG_EXECUTABLE "C:\\gstreamer\\1.0\\x86_64\\bin\\pkg-config.exe")
+    if(NOT DEFINED PKG_CONFIG_EXECUTABLE)
+      if(EXISTS "C:\\gstreamer\\1.0\\x86_64\\bin\\pkg-config.exe")
+        set(PKG_CONFIG_EXECUTABLE "C:\\gstreamer\\1.0\\x86_64\\bin\\pkg-config.exe")
+      endif()
+    elseif(DEFINED PKG_CONFIG_EXECUTABLE)
+      message(STATUS "Gstreamer pkg-config path set to ${PKG_CONFIG_EXECUTABLE}")
     else()
-      set(PKG_CONFIG_EXECUTABLE "D:\\gstreamer\\1.0\\x86_64\\bin\\pkg-config.exe")
+      message(FATAL_ERROR "Gstreamer not found in default path. Set the appropriate path with -DPKG_CONFIG_EXECUTABLE=<path>")
```

Comment:
Its a `FATAL_ERROR`. We error out. Since we do not have a gstreamer based control like we have in CPP, we should fail early instead of going through the ordeal.